server/client_manager.py

```diff
@@ -377,6 +379,32 @@ def gimp_message(self, message):
                        '((status?))']
             return random.choice(message)
 
+        def check_version(self, req_release, req_major, req_minor, exact):
```

Comment:
what if they're using a client that isn't AO2?

Reply:
Do we actually support that far back, and is there another client out there that isn't AO2?

That reminds me, it's out of scope here but we should really use this feature to add a minimum client version option so people aren't so out of date they're missing features.

Reply:
right, just remembered webclient, lemme go check what it's sending

Reply:
Webclient is sending the following:

``this.serv.send("ID#webAO#2.4.5#%");``

Reply:
I might change that...

Reply:
As long as the format stays the same I'll account for it.